store/store_test.js

```diff
@@ -44,9 +73,17 @@ Scenario('buy product', async ({ I, homePage, authPage, myAccountPage, productPa
     let totalAmount = await cartPage.getTotalAmount();
     console.log(totalAmount);
     I.assertEqual(numberTotalPrice, totalAmount, 'Prices are not in match');
+    let response = await I.sendGetRequest('https://bank.gov.ua/NBUStatService/v1/statdirectory/exchange?valcode=USD&json');
+    console.log(response.data[0].rate);
+    let uahAmount = (totalAmount * response.data[0].rate);
+    console.log(Math.round(uahAmount));
     cartPage.proceedToCheckout();
     let orderReference = await cartPage.getReferenceOrder();
     console.log(orderReference);
```

Comment:
add few words, when use `console.log()` to be more informative
![image](https://user-images.githubusercontent.com/19595492/188729899-63accfad-cf98-4d3d-b8c8-b3ad15ba96e6.png)